css/style.css

```diff
@@ -1,173 +1,173 @@
+  background-color: var(--main-blue);
+  display: flex;
+  align-items: center;
+  flex-direction: column;
+  justify-content: space-between;
+  padding: 2em 0;
 }
 
-#navbar a:hover {
-    background-color: var(--main-blue);
+.project-section > h1 {
+  border-bottom: 2px solid var(--main-white);
+  color: var(--main-white);
+  margin-top: 1em;
+  margin-bottom: 0.5em;
 }
 
-#welcome-section{
-    height: 100vh;
-    display: flex;
-    align-items: center;
-    justify-content: center;
-    flex-direction: column;
-    background-color: #000;
-    color: var(--main-white);
-    background-image: linear-gradient(62deg, #3a3d40 0%, #181719 100%);
-    font-size: 2.6em;
+.project-tile {
+  margin: 1em;
 }
 
-#welcome-section h3 {
-    color: var(--main-red);
-    font-style: italic;
+.projects {
+  display: flex;
+  justify-content: space-evenly;
+  text-align: center;
+  color: var(--main-white);
+  margin-bottom: 1rem;
 }
 
-/* project section */
+.project {
+  background-color: var(--main-gray);
+  padding: 0 0 1em 0;
+  border-radius: 3px;
+  margin: 1em;
+}
 
-#project-section {
-    background-color: var(--main-blue);
-    display: flex;
-    align-items: center;
-    flex-direction: column;
-    justify-content: space-between;
+.project a {
+  color: var(--main-white);
 }
 
-#project-section>h1 {
-    color: var(--main-white);
-    margin-top: 1em;
-    margin-bottom: .5em;
+.project h3 {
+  padding: 0.7em 0 0 0;
 }
 
-#project-section>h1 {
-    border-bottom: 2px solid var(--main-white);
+#navbar {
+  display: flex;
+  justify-content: flex-end;
+  background-color: var(--main-red);
+  height: 3.5em;
+  align-items: center;
+  position: fixed;
+  width: 100vw;
 }
 
-#projects {
-    display: flex;
-    justify-content: space-evenly;
-    text-align: center;
-    color: var(--main-white);
-    margin-bottom: 1rem;
+#navbar a {
+  padding: 0.65em 1em;
+  color: var(--main-white);
+  font-size: 1.4em;
 }
 
-#project {
-    background-color: var(--main-gray);
-    padding: 0 0 1em 0;
-    border-radius: 3px;
-    margin: 1em
+#navbar a:hover {
+  background-color: var(--main-blue);
 }
 
-#project a {
-    color: var(--main-white);
+#welcome-section {
+  height: 100vh;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  flex-direction: column;
+  background-color: #000;
+  color: var(--main-white);
+  background-image: linear-gradient(62deg, #3a3d40 0%, #181719 100%);
+  font-size: 2.6em;
 }
 
-#project h3 {
-    padding: .7em 0 0 0 ;
+#welcome-section h3 {
+  color: var(--main-red);
+  font-style: italic;
 }
 
 #show-button {
-    margin-bottom: 1em;
+  margin-bottom: 1em;
 }
 
 #show-button button {
-    padding: 1em;
-    background-color: var(--main-gray);
-    color: var(--main-white);
-
+  padding: 1em;
+  background-color: var(--main-gray);
+  color: var(--main-white);
 }
 
 img {
-    height: 400px;
-    width: 400px;
-    border-radius: 3px;
+  height: 400px;
+  width: 400px;
+  border-radius: 3px;
```

Comment:
You did a good job with CSS 💯 but I think you should try to stick to a unit, if you are using em, rem, px, or any unit, try to be consistent within that unit, so you won't have any units related problems or even any responsive problem 👍